packages/Collapsible/stories/examples/Variations.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { Story, Rule } from "storybook/assets/styles/common.styles";
+import Collapsible from "../../src";
+
+const dummyContent = (
+  <p>
+    <strong>Content</strong> – children of the &lt;Collapsible&gt; is hidden while the collapsible is collapsed, and
+    visible with it is expanded.
+  </p>
+);
+
+const collapsibleDefaultProps = {
+  a11yText: "collapsible section",
+  isDisabled: false,
+  label: "Click to show/hide the content",
+  iconAlign: "left",
+};
+
+const ExampleStory = () => {
+  const [isCollapsed, setIsCollapsed] = React.useState(false);
+  const [isCollapsed2, setIsCollapsed2] = React.useState(false);
+  const [isCollapsed3, setIsCollapsed3] = React.useState(false);
+  const [isCollapsed4, setIsCollapsed4] = React.useState(false);
+
+  return (
+    <Story>
+      <Collapsible
+        {...collapsibleDefaultProps}
+        isCollapsed={isCollapsed}
+        onClick={() => setIsCollapsed(prevCollapsed => !prevCollapsed)}
+      >
+        {dummyContent}
+      </Collapsible>
+      <Rule />
+      <Collapsible
```

Comment:
nit: if these are all going to be in one story, maybe a title/description so in storybook can see how it is different?

Reply:
I'm planning to tackle the styling of the `Variations` stories in a different task... I know that I will get super fussy if I try doing it here now, so I deliberately left it pretty minimal.